frontend/src/metabase/visualizations/echarts/cartesian/scatter/model.ts

```diff
@@ -0,0 +1,36 @@
+import type { RawSeries, RowValue } from "metabase-types/api";
```

Comment:
The `scatter` folder inside `cartesian` mixes the folder structure a bit. So the idea was to have a folder per chart type with two separate folders for the model and option (and maybe events). But the `scatter` is a chart type, and it contains both model and option so it is harder to distinct what is used where by looking at the structure